packages/vm/tests/api/runCall.spec.ts

```diff
@@ -217,7 +217,6 @@ tape('Ensure that Istanbul sstoreCleanRefundEIP2200 gas is applied correctly', a
   }
 
   const result = await vm.runCall(runCallArgs)
-  console.log(result.gasUsed, result.execResult.gasRefund)
   t.equal(result.gasUsed.toNumber(), 5812, 'gas used incorrect')
   t.equal(result.execResult.gasRefund!.toNumber(), 4200, 'gas refund incorrect')
```

Comment:
```suggestion
  t.ok(result.gasUsed.eqn(5812), 'gas used correct')
  t.ok(result.execResult.gasRefund!.eqn(4200), 'gas refund correct')
```

I've seen this fixed in multiple PRs already (one of mine, one of holger's) but neither have been merged in yet lol so let's finally fix it here today for merging